kits/igor/igor2DemoIntegrated.m

```diff
@@ -100,14 +99,16 @@ function igor2DemoIntegrated( cam_module )
 fbk = robotGroup.getNextFeedbackFull();
```

Comment:
Small thing -- we should check for the FW revision, and if it is < 15.2.0, print a warning or error about balance controller gains.

Reply:
Added a note about this to the README for now.  Made an issue to add it to the Igor code.  Since usually the demo will be done headless, maybe we use this as a chance to try printing to the Mobile I/O app?, or just set a color state and pause/continue?  Not sure the more I think about it :-(